src/intl/time-durations.js

```diff
@@ -0,0 +1,235 @@
+// @flow
+
+import moment from 'moment'
+import 'moment/locale/cs'
+import 'moment/locale/de'
+import 'moment/locale/es'
+import 'moment/locale/fr'
+import 'moment/locale/it'
+import 'moment/locale/ja'
+import 'moment/locale/ko'
+import 'moment/locale/pt-br'
+import 'moment/locale/zh-cn'
+
+import 'moment-duration-format'
+
+export function setupMomentTranslations (locale: string, defaultLocale: string) {
+  if (locale === 'aa') {
+    moment.defineLocale('aa', {})
+  }
+
+  const chosen = moment.locale([ locale, defaultLocale ])
+  console.log(`Locale being used by moment: ${chosen}`)
+
+  //
+  // Extend the moment locale object for moment-duration-format:
+  //   https://github.com/jsmreese/moment-duration-format#extending-moments-locale-object
+  //
+  const translations = require('./translated-time-durations.json')
+  if (translations[locale]) {
+    const t:{ [messageId: string]: string } = translations[locale]
+
+    moment.updateLocale(locale, {
+      durationLabelsStandard: {
```

Comment:
Are we sure that our supported languages have only 1 plural form? If not then we should be ready for more then one plural form. This is the case i.e. in Polish where different plural is used for 2-4 and for >= 5.
For more reading  check: https://en.wikipedia.org/wiki/Dual_(grammatical_number)

Reply:
So basically the solution implemented isn't perfect but should be good enough.

The perfect solution would be to have moment-duration-format defined localizations the same way as moment does it. Moment embeds both the strings and functions on how to render plurals properly per locale.  Since I don't know that for different languages, and since we use zanata for translations, bridging them how I did I think is reasonable.

Also note: In Japanese, I'm fairly sure, they don't use plurals in this context at all.  So their translations for 1 second and >1 second will be the same.